src/storage/config.rs

```diff
@@ -23,7 +23,11 @@ const DEFAULT_GC_RATIO_THRESHOLD: f64 = 1.1;
 const DEFAULT_SCHED_CAPACITY: usize = 10240;
 const DEFAULT_SCHED_MSG_PER_TICK: usize = 1024;
 const DEFAULT_SCHED_CONCURRENCY: usize = 102400;
-const DEFAULT_SCHED_TOO_BUSY_THRESHOLD: usize = 4000;
+
+// Assume one TiKV node can write 100_000 kvs per second,
+// and write requests' average process time is about 100ms,
+// so here use 10_000 as the default value.
+const DEFAULT_SCHED_TOO_BUSY_THRESHOLD: usize = 10_000;
```

Comment:
Don't quite understand the explanation.

Reply:
Same too.

Reply:
You can imaging that the throughput of a pipe equals crosscut area of the pipe multiple the speed of water. In here the speed of water is same as how fast a request can process, the pipe size is same as the threshold above.

Reply:
https://en.wikipedia.org/wiki/Little%27s_law

Reply:
I think the threshold may be a little low if we consider KV count here.  100ms write seems too long.

Reply:
You misunderstood it, the faster request can be handled, the less kv can be piled up.